src/test/testdata/lsl/charLiterals.lsl

```diff
@@ -0,0 +1,13 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias Char = char;
+
+automaton A : Int {
+    var ch1: Char = '\u0041';
```

Comment:
I suggest adding an example of a more complex Unicode character here (and seeing the issues that may arise from that).
Like `\u72D0` for example.

Reply:
Was added

Reply:
But what about non-printable characters? Like `new-line` and `caret-return`? `bell-character`? `null-character`?